AdministratorAdministration+config.json

```diff
@@ -11,44 +11,258 @@
+                  }
+                }
+              }
+            },
+            {
+              "uuid": "aa-2-0-0-response-p-005",
+              "profile-name": "response-profile-1-0:PROFILE_NAME_TYPE_GENERIC_RESPONSE_PROFILE",
+              "response-profile-1-0:response-profile-pac": {
+                "response-profile-capability": {
+                  "operation-name": "/v1/inform-about-application-in-generic-representation",
+                  "field-name": {
+                    "static-field-name": "ownerName"
+                  },
+                  "description": "Name of the ApplicationOwner",
+                  "datatype": "string"
+                },
+                "response-profile-configuration": {
+                  "value": {
+                    "value-reference": "/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-name"
+                  }
+                }
+              }
+            },
+            {
+              "uuid": "aa-2-0-0-response-p-006",
+              "profile-name": "response-profile-1-0:PROFILE_NAME_TYPE_GENERIC_RESPONSE_PROFILE",
+              "response-profile-1-0:response-profile-pac": {
+                "response-profile-capability": {
+                  "operation-name": "/v1/inform-about-application-in-generic-representation",
+                  "field-name": {
+                    "static-field-name": "ownerEmailAddress"
+                  },
+                  "description": "Email address of the ApplicationOwner",
+                  "datatype": "string"
+                },
+                "response-profile-configuration": {
+                  "value": {
+                    "value-reference": "/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-email-address"
+                  }
+                }
+              }
+            },
+            {
+              "uuid": "aa-2-0-0-response-p-007",
+              "profile-name": "response-profile-1-0:PROFILE_NAME_TYPE_GENERIC_RESPONSE_PROFILE",
+              "response-profile-1-0:response-profile-pac": {
+                "response-profile-capability": {
+                  "operation-name": "/v1/inform-about-release-history-in-generic-representation",
+                  "field-name": {
+                    "field-name-reference": "/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-list=0/release-number"
+                  },
+                  "description": "List of registered application names and release numbers",
```

Comment:
description in line 224 and 243 (GenericResponseProfile :: /v1/inform-about-release-history-in-generic-representation :: description) to be changed to "List of release numbers and changes"

Reply:
224: Took the value from the ApplicationPattern ("Represents the first entry in the list of releases"). Many thanks for indicating.
243: Changed as proposed.